chore(docs): clarify that variables be edited through the web UI (#15799)
coder/coder: docs/admin/templates/extending-templates/variables.md

1. _Manual input in CLI_: You can manually input values for Terraform variables
   directly in the CLI during the deployment process.

> 2. _Web UI_: You can set and/or edit variable values under "Variables" in the template's settings.   

Reviewer: ```suggestion
1. _Web UI_: You can set or edit variable values under **Variables** in the template's settings.   
```

> 3. _Command-line argument_: Utilize the `--var name=value` command-line argument

Reviewer: ```suggestion
1. _Command-line argument_: Utilize the `--var name=value` command-line argument
```

   to specify variable values inline as key-value pairs.

> 4. _Variables file selection_: Alternatively, you can use a variables file

Reviewer: ```suggestion
1. _Variables file selection_: Alternatively, you can use a variables file
```
